packages/fluentui/react-builder/src/components/Knobs.tsx

```diff
@@ -139,47 +140,85 @@ type DesignKnobProps = {
     value: number;
   }) => void;
   onPropDelete: ({ jsonTreeElement, name }: { jsonTreeElement: JSONTreeElement; name: string }) => void;
+  onPropNavigate: ({ jsonTreeElement, path }: { jsonTreeElement: JSONTreeElement; path: string[] }) => void;
   info: ComponentInfo;
   jsonTreeElement: JSONTreeElement;
+  propPath: string[];
+  getShorthandComponentInfo: (componentType: string) => ComponentInfo;
 };
 
 const isHandledType = (type: string): boolean => {
-  return ['boolean', 'string', 'literal', 'React.ElementType', 'number'].includes(type);
+  return ['boolean', 'string', 'literal', 'React.ElementType', 'number', 'ShorthandValue'].includes(type);
 };
 
 export const Knobs: React.FunctionComponent<DesignKnobProps> = ({
   onPropChange,
   onPropDelete,
+  onPropNavigate,
   info,
   jsonTreeElement,
+  propPath,
+  getShorthandComponentInfo,
 }) => {
   const [menuActivePane, setMenuActivePane] = React.useState<'props' | 'accessibility'>('props');
-  const getValues = React.useCallback(
-    prop => {
-      const propValue = jsonTreeElement.props?.[prop.name];
-      const tempTypes = _.uniq(_.map(prop.types, 'name'));
-      const types = isHandledType(tempTypes[0]) ? tempTypes : _.uniq(_.map(prop.resolvedType, 'name'));
-      const isLiteral = _.every(types, name => name === 'literal');
-      // console.log(prop.name, types, prop);
-      const options = isLiteral
-        ? _.map(Array.isArray(prop.resolvedType) ? prop.resolvedType : prop.types, 'value')
-        : null;
+  const getValues = React.useCallback((obj, prop) => {
+    const propValue = obj.props?.[prop.name];
+    const tempTypes = _.uniq(_.map(prop.types, 'name'));
+    const types = isHandledType(tempTypes[0]) ? tempTypes : _.uniq(_.map(prop.resolvedType, 'name'));
+    const isShorthand = tempTypes[0] === 'ShorthandValue';
+    const shorthandType =
+      isShorthand && _.find(prop.resolvedType.types, t => t.type === 'intersection')['types'][0].name;
+    const isLiteral = _.every(types, name => name === 'literal');
+    // console.log(prop.name, types, prop);
+    const options = isLiteral
+      ? _.map(Array.isArray(prop.resolvedType) ? prop.resolvedType : prop.types, 'value')
+      : isShorthand
+      ? [shorthandType]
+      : null;
```

Comment:
I found it more legible not having more than 1 ternary operator... But maybe it's just me so it would be more like __nit__

```suggestion
    let options = null;
    if (isLiteral) {
        options = _.map(Array.isArray(prop.resolvedType) ? prop.resolvedType : prop.types, 'value');
    } else if (isShorthand)  {
        options = [shorthandType]
    }
```